games/game_bladeandsorcery.py

```diff
@@ -1,18 +1,198 @@
+        self._label.setText("Character")
+
+        self._formLayout.setWidget(0, QFormLayout.ItemRole.LabelRole, self._label)
+
+        self._label_2 = QLabel()
+        self._label_2.setObjectName("label_2")
+        self._label_2.setFont(font)
+        self._label_2.setText("Game Mode")
+
+        self._formLayout.setWidget(1, QFormLayout.ItemRole.LabelRole, self._label_2)
+
+        self._label_3 = QLabel()
+        self._label_3.setObjectName("label_4")
+        self._label_3.setFont(font)
+        self._label_3.setText("Created At")
+
+        self._formLayout.setWidget(2, QFormLayout.ItemRole.LabelRole, self._label_3)
+
+        self._label_4 = QLabel()
+        self._label_4.setObjectName("label_4")
+        self._label_4.setFont(font)
+        self._label_4.setText("Last Saved")
+
+        self._formLayout.setWidget(3, QFormLayout.ItemRole.LabelRole, self._label_4)
+
+        self._label_5 = QLabel()
+        self._label_5.setObjectName("label_3")
+        self._label_5.setFont(font)
+        self._label_5.setText("Session Duration")
 
+        self._formLayout.setWidget(4, QFormLayout.ItemRole.LabelRole, self._label_5)
+
+        font1 = QFont()
+        font1.setBold(True)
+
+        self._characterLabel = QLabel()
+        self._characterLabel.setObjectName("characterLabel")
+        self._characterLabel.setFont(font1)
+        self._characterLabel.setText("")
+
+        self._formLayout.setWidget(0, QFormLayout.ItemRole.FieldRole, self._characterLabel)
+
+        self._gameModeLabel = QLabel()
+        self._gameModeLabel.setObjectName("gameModeLabel")
+        self._gameModeLabel.setFont(font1)
+        self._gameModeLabel.setText("")
+
+        self._formLayout.setWidget(1, QFormLayout.ItemRole.FieldRole, self._gameModeLabel)
+
+        self._dateLabel = QLabel()
+        self._dateLabel.setObjectName("dateLabel")
+        self._dateLabel.setFont(font1)
+        self._dateLabel.setText("")
+
+        self._formLayout.setWidget(2, QFormLayout.ItemRole.FieldRole, self._dateLabel)
+
+        self._sessionLabel = QLabel()
+        self._sessionLabel.setObjectName("sessionLabel")
+        self._sessionLabel.setFont(font1)
+        self._sessionLabel.setText("")
+
+        self._formLayout.setWidget(3, QFormLayout.ItemRole.FieldRole, self._sessionLabel)
+
+        self._elapsedTimeLabel = QLabel()
+        self._elapsedTimeLabel.setObjectName("elapsedTimeLabel")
+        self._elapsedTimeLabel.setFont(font1)
+        self._elapsedTimeLabel.setText("")
+
+        self._formLayout.setWidget(4, QFormLayout.ItemRole.FieldRole, self._elapsedTimeLabel)
+
+        self._verticalLayout.addLayout(self._formLayout)
+
+        self.setLayout(self._verticalLayout)
+        self.setWindowFlags(Qt.WindowType.ToolTip | Qt.WindowType.BypassGraphicsProxyWidget)
+        self.setWindowOpacity(
+            self.style().styleHint(QStyle.StyleHint.SH_ToolTipLabel_Opacity) / 255.0
+        )
+
+    def setSave(self, save: BaSSaveGame):
```

Comment:
Does this pass linting? Not sure if you can change the argument type in an override method. If it does not, you can simply assert the type at the beginning of the method.

Reply:
It seems to.